src/appfl/config/config.py

```diff
@@ -70,6 +70,8 @@ class Config:
     logginginfo: DictConfig = OmegaConf.create({})
     summary_file: str = ""
 
+    # GPU configuration
+    num_gpu: int = 1
```

Comment:
Is this parameter used for the runs with mpi? Can this be used with gRPC?

Reply:
Yes, this parameter is independent of the MPI processes. The parameter is used only if the CUDA is available according to the torch.cuda.is_available() function. If the CUDA is available, the default config is set to 1, and replaced by torch.cuda.device_count() function.